boards/arm/96b_argonkey/doc/96b_argonkey.rst

```diff
@@ -0,0 +1,164 @@
+to run on these boards.
+
+This board acts as a sensor hub platform for all 96Boards compliant
+family products. It can also be used as a standalone board.
+
+.. figure:: img/96b-argonkey-front.jpg
+     :width: 557px
+     :align: center
+     :height: 480px
+     :alt: 96Boards Argonkey
+
+     96Boards Argonkey
+
+Hardware
+********
+
+96Boards Argonkey provides the following hardware components:
+
+- STM32F412CG in UFQFPN48 package
+- ARM |reg| 32-bit Cortex |reg|-M4 CPU with FPU
+- 100 MHz max CPU frequency
+- 1.8V work voltage
+- 1024 KB Flash
+- 256 KB SRAM
+- On board sensors:
+
+  - Humidity: ST Micro HTS221
+  - Temperature/Pressure: ST Micro LPS22HB
+  - ALS: Intersil ISL29034
+  - Proximity: STMicro VL53L0X
+  - Acclerometer/Gyroscope: STMicro LSM6DSL
+  - Geomagnetic: STMicro LIS2MDL
+  - AMR Hall sensor: MRMS501A
+  - Microphone: STMicro MP34DT05
+
+- 2 User LEDs
+- 16 General purpose LEDs
+- GPIO with external interrupt capability
+- UART
+- I2C (3)
+- SPI (1)
+- I2S (1)
+
+Supported Features
+==================
+
+The Zephyr 96b_argonkey board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | system clock                        |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash                               |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+
+More details about the board can be found at `96Boards website`_.
+
+The default board configuration can be found in the defconfig file:
+
+        ``boards/arm/96b_argonkey/96b_argonkey_defconfig``
+
+Connections and IOs
+===================
+
+LED
+---
+
+- LED1 / User1 LED = PB2
+- LED2 / User2 LED = PC13
+
+Push buttons
+------------
+
+- BUTTON = RST (BT1)
+- BUTTON = USR (BT2)
+
+System Clock
+============
+
+96Boards Argonkey can be driven by an internal oscillator as well as the main
+PLL clock. By default System clock is sourced by PLL clock at 84MHz, driven
```

Comment:
the system clock
    the PLL clock
    the internal oscillator

Reply:
need to specify all of this? ok